files/usr.bin.pollen

```diff
@@ -0,0 +1,28 @@
+# vim:syntax=apparmor
+# Created for Ubuntu by: Dustin Kirkland <kirkland@ubuntu.com>
+# ------------------------------------------------------------------
+#
+#    Copyright (C) 2014 Dustin Kirkland
+#    Copyright (C) 2014 Canonical Ltd.
+#
+#    This program is free software; you can redistribute it and/or
+#    modify it under the terms of version 3 of the GNU General Public
+#    License published by the Free Software Foundation.
+#
+# ------------------------------------------------------------------
+#include <tunables/global>
+/usr/bin/pollen {
+  #include <abstractions/base>
+  #include <abstractions/nameservice>
+  capability net_bind_service,
+  /dev/random rw,
+  /dev/urandom rw,
+  /etc/pollen/* r,
+  /proc/sys/net/core/somaxconn r,
+  /proc/sys/kernel/hostname r,
+  /proc/sys/kernel/random/entropy_avail r,
+  /usr/bin/pollen rm,
+  /dev/pts/[0-9]* rw,
+  # Site-specific additions and overrides. See local/README for details.
+  #include <local/usr.bin.pollen>
+}
```

Comment:
Missing line at EOF

Reply:
Still seems to be an issue, perhaps if this won't get resolved leave a note why not?

Reply:
sorry, I forgot to upload the changes yesterday before EOD